python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -88,7 +91,9 @@ def asset(
     name: Optional[str] = None,
     key_prefix: Optional[CoercibleToAssetKeyPrefix] = None,
     ins: Optional[Mapping[str, AssetIn]] = None,
-    non_argument_deps: Optional[Union[Set[AssetKey], Set[str]]] = None,
+    deps: Optional[
+        Sequence[Union[CoercibleToAssetKey, AssetsDefinition, SourceAsset]]
+    ] = None,  # if deps will become the default experience, should it go above ins?
```

Comment:
I think we can omit this comment. The ordering doesn't really matter because of the wildcard.